src/devtools-connector/tests/arc-stores-fetcher-test.ts

```diff
@@ -30,7 +32,7 @@ describe('ArcStoresFetcher', () => {
     const runtime = new Runtime(new StubLoader({}), FakeSlotComposer, context);
     const arc = runtime.newArc('demo', storageKeyPrefixForTest(), {inspectorFactory: devtoolsArcInspectorFactory});
 
-    const foo = arc.context.findSchemaByName('Foo').entityClass();
+    const foo = Entity.createEntityClass(arc.context.findSchemaByName('Foo'), null);
```

Comment:
Might be worth having a helper testing function that has a comment explaining the passing of the 'null' context. It makes sense in cases like this (to pass null) but is a bit surprising reading the tests.

Reply:
Again, no idea. I just replaced a call to a Schema method with the one line it contained, in order to remove the dependency from Schema.